pkg/install/resources.go

```diff
@@ -37,7 +37,13 @@ func imageVersion() string {
 }
 
 // DefaultImage is the default image to use for the Velero deployment and restic daemonset containers.
-var DefaultImage = "gcr.io/heptio-images/velero:" + imageVersion()
+var (
+	DefaultImage               = "gcr.io/heptio-images/velero:" + imageVersion()
+	DefaultVeleroPodCPURequest = "100m"
```

Comment:
Is there a reason we want to define these defaults here instead of simply hardcoding them in the install command?

Reply:
I don't remember exactly why I put the DefaultImage here. I think it may have been either referenced elsewhere in the codebase and I consolidated it here, or I was thinking external consumers of the install package could use it without relying on the CLI.

Reply:
Looks like it's only being used in the install command package right now. I guess it could be useful for external consumers, though.

Reply:
I think it makes sense to leave it where it is given the current package structure.